mava/adders/reverb/episode.py

```diff
@@ -29,25 +29,24 @@
 import tree
 from acme import specs, types
 from acme.adders.reverb import utils
-from acme.utils import tree_utils
```

Comment:
Did you upgrade the episode adder and just use tests to determine if it is probably working? We don't have any system implementation using the episode adder correct?

Reply:
Yeah, I just wrote the tests to check the adder works as expected.